frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/index.tsx

```diff
@@ -65,7 +71,8 @@ const DatasetRow: FC<Props> = ({
 
   const [uploadProgress, setUploadProgress] = useState(upload_progress);
 
-  if (queryResult.isError) console.error(queryResult.error);
+  if (queryResult.isError && !(queryResult.error instanceof CancelledError))
```

Comment:
For readability, if the condition and its block span over one line, can we add curly braces please?

E.g.,

```ts
if (queryResult.isError && !(queryResult.error instanceof CancelledError)) {
  console.error(queryResult.error);
}
```